.ci/smoke/smoke-tests-linux.yml

```
@@ -15,6 +15,7 @@ jobs:
   - job: Linux
     pool:
       name: 'React Native smoke tests - Linux'
+    timeoutInMinutes: 25
```

Will this timeout be enough? Could it be better to set it up at least on 30 min?

---

In the most cases it was enough, but it will make sense when we add more new tests
```suggestion
    timeoutInMinutes: 30
```